SECURITY.md

```diff
@@ -0,0 +1,37 @@
+# Security Policy
+
+## Supported Versions
+
+Security-related reports are considered for official numbered releases
+starting with v3.5. However, issues that do not affect the current Stable or
+Beta series are unlikely to be fixed. Please see
+http://www.squid-cache.org/Versions/ for the list of releases that belong to
+the current series.
+
+Reports about security issues in the Development series are welcomed. However,
+development series contains experimental code that does not qualify for CVE
+allocation under MITRE policy.
+
+
+## Reporting a Vulnerability
+
+To report security-sensitive bugs, please post to the squid-bugs mailing
+(list)[http://www.squid-cache.org/Support/mailing-lists.html#squid-bugs]. It
+is a closed list (although anyone can post), and security related bug reports
+are treated in confidence until the impact has been established.
```

Comment:
Should we add "at least" here? It seems odd to imply that confidentiality is gone once the impact is known...

```suggestion
is a closed list (although anyone can post), and security related bug reports
are treated in confidence at least until the impact has been established.
```